scripts/Dockerfile-airflow

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM apache/airflow
+
+ARG LIMINAL_VERSION='apache-liminal'
+ARG DAG_FOLDER='/opt/airflow/dags/'
+ADD scripts/* ${DAG_FOLDER}
```

Comment:
Do we need all these files or just `requirements.txt`?

Reply:
need them to be able to install local liminal versions inside the docker